docs/pages/docs/_meta.ts

```diff
@@ -30,4 +30,5 @@ export default {
   '*': {
     title: ''
   },
+  foundation: 'Foundation',
```

Comment:
It shouldn't be below the wildcard imo. The wildcard generally shouldn't be used and it just a fallback just in case.